src/ecdsa/ecdh.py

```diff
@@ -169,47 +211,58 @@ def load_received_public_key(self, public_key):
 
     def load_received_public_key_bytes(self, public_key_str):
         """
-        Load public key from bytes string.
+        Load public key from byte string.
         Uses current curve and checks if key length corresponds to
         the current curve.
+
         :param public_key_str: public key in bytes string format
+        :type public_key_str: byte string
         :return: none
         """
         return self.load_received_public_key(
```

Comment:
if `load_received_public_key` returns None (doesn't return anything), why do we explicitly return it?

Reply:
that's still unfixed

Reply:
fixed